DEFAULT_CONFIG.json5

```diff
@@ -30,6 +30,8 @@
     /// The list of endpoints to connect to.
     /// Accepts a single list (e.g. endpoints: ["tcp/10.10.10.10:7447", "tcp/11.11.11.11:7447"])
     /// or different lists for router, peer and client (e.g. endpoints: { router: ["tcp/10.10.10.10:7447"], peer: ["tcp/11.11.11.11:7447"] }).
+    ///
+    /// See https://docs.rs/zenoh/latest/zenoh/prelude/struct.EndPoint.html
```

Comment:
The link will need to be updated according to the reorganisation introduced in `1.0.0-beta.3`

Reply:
OK. Although "latest" will keep pointing to 0.11 until 1.0 is released and the links will remain broken.